src/api/IBMiContent.ts

```diff
@@ -871,27 +871,31 @@ export default class IBMiContent {
   }
 
   async memberResolve(member: string, files: QsysPath[]): Promise<IBMiMember | undefined> {
-    const inAmerican = (s: string) => { return this.ibmi.sysNameInAmerican(s) };
-    const inLocal = (s: string) => { return this.ibmi.sysNameInLocal(s) };
+    const localVariants = this.ibmi.variantChars;
+    const forSystem = (s: string) => { return this.ibmi.sysNameInAmerican(s, this.ibmi.qsysPosixPathsRequireTranslation) };
+    const fromSystem = (s: string) => { return this.ibmi.qsysPosixPathsRequireTranslation ? this.ibmi.sysNameInLocal(s) : s };
 
     // Escape names for shell
     const pathList = files
       .map(file => {
         const asp = file.asp || this.ibmi.getCurrentIAspName();
         if (asp && asp.length > 0) {
           return [
-            Tools.qualifyPath(inAmerican(file.library), inAmerican(file.name), inAmerican(member), asp, true),
-            Tools.qualifyPath(inAmerican(file.library), inAmerican(file.name), inAmerican(member), undefined, true)
+            Tools.qualifyPath(forSystem(file.library), forSystem(file.name), forSystem(member), asp, localVariants),
+            Tools.qualifyPath(fromSystem(file.library), fromSystem(file.name), fromSystem(member), undefined, localVariants)
```

Comment:
It used to be `inAmerican`, so I think this should be `forSystem` here.
```suggestion
            Tools.qualifyPath(fromSystem(file.library), forSystem(file.name), forSystem(member), undefined, localVariants)
```